Tools/Scripts/generate-dirty-files

```diff
@@ -86,9 +98,10 @@ def parse_results_file(args, file_path):
     return None, None, None, None
 
 
-def find_project_results(args, project, file_list, results_data, issue_to_report):
+def find_project_results(args, project, file_list, results_data, issue_to_report, results_for_upload):
     bug_counts = {}
     files_per_project = {key: set() for key in CHECKER_MAP.values()}
+    results_for_upload[project] = {}
```

Comment:
Instead of doing this, why don't we just have a `project_results_for_upload = {}` and then return that, and then at the callsite set `results_for_upload[project]` equal to what we returned?

Reply:
Yeah, I like this better. Thanks!